### PR TITLE
tools/ebpf: fix the setup of the VM

### PR DESCRIPTION
**Pull request by sfluor**
### What does this PR do?

Fix the dev setup for the ebpf devenv

### CODE REVIEW

File: tools/ebpf/dev_setup.sh [hunk 1]
```diff
@@ -3,74 +3,118 @@
...
+- \`curl_agent\` http://unix/debug/stats to query endpoints on the network-tracer
+----------------------------------------------"' >> ~/.profile
 EOD
 
 # necessary to get group membership to be respected
-VAGRANT_CWD=$(pwd) vagrant reload
+vagrant reload
 
-echo "your development environment is ready; use \`VAGRANT_CWD=$cwd vagrant ssh\` to ssh in"
+echo "your development environment is ready; use \`grant ssh\` to ssh in"
```
Comment (leeavital) [inline tools/ebpf/dev_setup.sh]: typo: `grant` should be `vagrant`

File: tools/ebpf/dev_setup.sh [hunk 2]
```diff
@@ -3,74 +3,118 @@
...
+sudo apt-get install -y clang-8 llvm-8
+sudo ln -sf /usr/bin/clang-8 /usr/bin/clang
+sudo ln -sf /usr/bin/llc-8 /usr/bin/llc
+EOD
 
-# install invoke
+echo "installing linux-headers"
 cat <<EOD | vagrant ssh
-sudo pip install invoke
+sudo apt-get install -y linux-headers-\$(uname -r)
```
Comment (shang-wang) [inline tools/ebpf/dev_setup.sh]: I don't think the `\` is suppose to be there right?

File: tools/ebpf/dev_setup.sh [hunk 3]
```diff
@@ -3,74 +3,118 @@
...
+  enabled: true
+  log_file: "/home/vagrant/network-tracer.log"
+  bpf_debug: true
+" | sudo tee /etc/datadog-agent/network-tracer.yaml
+EOD
+
+echo "Adding some aliases and login help"
 cat <<EOD | vagrant ssh
-sudo apt-get install -y clang-format
+echo 'alias goforit="cd /home/vagrant/go/src/github.com/DataDog/datadog-agent"' >> ~/.bashrc
```
Comment (shang-wang) [inline tools/ebpf/dev_setup.sh]: 😆

**Review by leeavital — DISMISSED** [review]

**Review by shang-wang — DISMISSED** [review]
Just one question but otherwise 👍

File: tools/ebpf/dev_setup.sh (hunk 2 above)
Comment (sfluor) [reply tools/ebpf/dev_setup.sh]: It's to avoid the expression to be evaluated on the computer running the VM

File: tools/ebpf/dev_setup.sh (hunk 3 above)
Comment (sfluor) [reply tools/ebpf/dev_setup.sh]: I had to 😄

**Review by shang-wang — APPROVED** [review]

**Pull request merged**